src/Bard.py

```diff
@@ -310,14 +317,14 @@ async def ask(self, message: str) -> dict:
         sys.exit(0)
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "--session",
-        help="__Secure-1PSID cookie.",
+        "--session --session_ts",
+        help="__Secure-1PSID cookie and __Secure_1PSIDTS cookie.",
```

Comment:
You removed the --session argument? Does this work?

Reply:
done
<img width="245" alt="image" src="https://github.com/acheong08/Bard/assets/69547456/2686dc95-19de-435e-83ee-59a7f7589551">

<img width="1227" alt="image" src="https://github.com/acheong08/Bard/assets/69547456/9628349c-b84b-4310-b9c3-8f58b4e6d254">

Reply:
#71